specification/threatmodel.adoc

```diff
@@ -1,187 +1,300 @@
 [[threatmodel]]
 === Adversary Model
 
-_Unprivileged Software adversary -_ This includes software executing in 
-U-mode managed by S/HS/M-mode system software. This adversary can access 
-U-mode CSRs, process/task memory, CPU registers in the process context 
+_Unprivileged Software adversary -_ This includes software executing in
+U-mode managed by S/HS/M-mode system software. This adversary can access
+U-mode CSRs, process/task memory, CPU registers in the process context
 managed by system software.
 
-_System Software adversary_ - This includes system software executing in 
-S/HS/VS modes. Such an adversary can access S/HS/VS privileged CSRs, 
+_System Software adversary_ - This includes system software executing in
+S/HS/VS modes. Such an adversary can access S/HS/VS privileged CSRs,
 assigned system memory, CPU registers and IO devices.
 
-_Startup Software adversary_ - This includes system software executing in 
-early/boot phases of the system (in M-mode), including BIOS, memory 
-configuration code, device option ROM/firmware that can access system 
+_Startup Software adversary_ - This includes system software executing in
+early/boot phases of the system (in M-mode), including BIOS, memory
+configuration code, device option ROM/firmware that can access system
 memory, CPU registers, IO devices and IOMMU etc.
 
```

Comment:
and platform configuration registers (e.g., address range decoders, SoC fabric configuration, etc.).

Reply:
ack